python/tests/test_transforms.py

```diff
@@ -139,6 +143,104 @@ def test_string_with_surrogate_pair():
     assert bucket_transform(string_with_surrogate_pair) == mmh3.hash(as_bytes)
 
 
+@pytest.mark.parametrize(
+    "date,date_transform,expected",
+    [
+        (47, YearTransform(), "2017"),
+        (575, MonthTransform(), "2017-12"),
+        (17501, DayTransform(), "2017-12-01"),
+    ],
+)
+def test_date_to_human_string(date, date_transform, expected):
+    assert date_transform.to_human_string(DateType(), None) == "null"
+    assert date_transform.to_human_string(DateType(), date) == expected
+
+
+@pytest.mark.parametrize(
+    "timestamp,time_transform,expected",
```

Comment:
The first value is not a timestamp, it is the result of applying a transform. The first 3 cases here are identical to the cases for "date" because of that. Can you remove the `date` version of this and rename this argument to "value"? The test name should also not include "timestamp" or "ts" because this is converting output values to strings.

Reply:
👌  updated.